lessonbuilder/tool/src/java/org/sakaiproject/lessonbuildertool/tool/beans/SimplePageBean.java

```diff
@@ -393,6 +402,8 @@
 			"Teal",
 			"Purple"
 	};
+	public static final String pageLayoutValues[] = {"none", "addSubpageList", "interiorResources", "interiorTask"};
+	public static final String subpageCountValues[] = {"1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20"};
```

Comment:
Not a big fan of this, could we maybe use something like IntStream.range(1, 21).toArray()

Reply:
I have remade the array where it gets used, in ShowPageProducer.createPageLayoutTab, with a loop if that works better.